great_expectations/rule_based_profiler/parameter_builder/value_set_multi_batch_parameter_builder.py

```diff
@@ -153,5 +156,11 @@ def _get_unique_values_from_nested_collection_of_sets(
     """
 
     flattened: List[Set[Any]] = list(itertools.chain.from_iterable(collection))
-    unique_values: Set[Any] = set().union(*flattened)
+    unique_values: Set[Any] = set(
+        filter(
+            lambda element: element is not None,
+            set().union(*flattened),
+        )
+    )
+
```

Comment:
This is some beautiful code but a little hard to understand. Can we simplify it at all? Or perhaps add a comment?

Reply:
@cdkini I opted for the comment.  Thanks! :)